### DISCUSSION TITLE
Replacing ansible repository for Sympa at Projects related to Sympa

### DISCUSSION THREAD

**Discussion by racke** [sympa-community/sympa#1634] [Team Posts]
The code on https://github.com/sympa-community/sympa-ansible is not on development anymore. It also doesn't focus on Sympa itself.

I suggest to replace that with my Sympa Ansible role at https://github.com/racke/ansible-role-sympa.

**Comment by ikedas** [comment]
I feel replacing repository is good idea :+1:.

**Comment by racke** [comment]
Archived  https://github.com/sympa-community/sympa-ansible and adjusted the pinning.

**Comment by ikedas** [comment]
Thanks!
